### PR TITLE
Add new devops recipe to the Accelerate pages

### PR DESCRIPTION
**Pull request by GuitarRich**
## Description / Motivation

Add the DevOps recipe

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update (non-breaking change; modified files are limited to the `/data` directory or other markdown files)

## Checklist:

- [x] I have read the Contributing guide.
- [x] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change.
- [x] My change is a documentation change and there are NO other updates required.
- [ ] My change has new or updated images which are stored in the `/public/images` folder that need to be migrated to Sitecore DAM

### CODE REVIEW

File: apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md [hunk 1]
```diff
@@ -1,11 +1,129 @@
...
+
+- You’re not using Vercel to host your head.
+- You’re not using a Monorepo and your head and XM Cloud application elements are stored in separate repositories.
+- You’re using the CLI to deploy XM Cloud as described above, and want to manage your head deployments using the CI/CD tooling.
+
+### Deploy using the Vercel Connector
+
+If you can use the Vercel Connector then once you have provisioned your XM Cloud Infrastructure and created a Site return to the Sites tab in the Deploy application for your Environment, and click the button on the right and choose “Set up hosting”. Complete the sections and follow the prompts to deploy your site to a new Vercel installation
```
Comment (crhistianramirez) [inline apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md:86]: The first part seems unnecessary, especially given the section title, and it feels a bit long for a sentence. Recommended change:

Once you have provisioned your XM Cloud Infrastructure and created a Site return to the Sites tab in the Deploy application for your Environment. Then, click the button on the right and choose “Set up hosting”. Complete the sections and follow the prompts to deploy your site to a new Vercel installation

File: apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md [hunk 2]
```diff
@@ -1,11 +1,129 @@
...
+## Solution
+
+### Provisioning the XM Cloud Infrastructure
+
+When deploying the XM Cloud infrastructure elements, there are two options for how to go about this:
+
+1. Use the inbuilt XM Cloud Deploy Application and its GitOps approach
+1. Roll your own CI CD pipeline using your DevOps tooling of choice and the Sitecore CLI
+
+There are a few factors that can help you decide on the best approach for your use case. By default customers are recommended to use the provided XM Cloud Deploy application. This will reduces the client's Time to Market, and means they automatically receive the benefits as the tooling is improved.
```
[the remark this answers is not in the document]
Comment (GuitarRich) [reply apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md:22]: Fixed

File: apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md [hunk 3]
```diff
@@ -1,11 +1,129 @@
...
+
+However, there are some scenarios where that might not be possible, some examples of these are:
+
+- Currently GitHub and Azure DevOps are the only two supported source control systems. If you’re using a different source control provider, then you won’t be able to use the Deploy Application
```
[the remark this answers is not in the document]
Comment (GuitarRich) [reply apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md:26]: Fixed

File: apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (GuitarRich) [reply apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md:22]: Fixed

File: apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md [hunk 4]
```diff
@@ -1,11 +1,129 @@
...
+If you can use the Vercel Connector then once you have provisioned your XM Cloud Infrastructure and created a Site return to the Sites tab in the Deploy application for your Environment, and click the button on the right and choose “Set up hosting”. Complete the sections and follow the prompts to deploy your site to a new Vercel installation
+
+<Alert>
+  <AlertIcon />
+  You can learn more about how to manage your Vercel Connector on our Documentation Site: https://doc.sitecore.com/xmc/en/developers/xm-cloud/manage-connections-for-source-control-and-hosting-providers.html
+</Alert>
+
+### Deploy a custom head Deployment
+
+If you need to create a custom deployment for your head, then the approach you take will vary for each project, depending on the Hosting Platform and CI/CD tooling that you choose to use. You can see some getting started guides for some of the more common hosting providers helow:
```
[the remark this answers is not in the document]
Comment (GuitarRich) [reply apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md:95]: Fixed

**Review by crhistianramirez — APPROVED** [review]

**Review by crhistianramirez — APPROVED** [review]

**Pull request merged**